Majasdarbs/09-majasdarbs/forStudents/GvidoZarins/app.js

```diff
@@ -0,0 +1,32 @@
+const dogSelector = document.getElementById("dogSelector");
+const dogImg = document.getElementById("dogImg");
+
+// Render list and image
+const renderDogData = (data) => {
+   if (Array.isArray(data)) {
+    data.forEach((elem) => {
+        const o = document.createElement("option");
+        o.setAttribute("value", elem.value);
+        o.innerHTML = `${elem.name}`;
+        dogSelector.appendChild(o);   
+    });
+   }
+
+    dogImg.innerHTML = `<img src="${data.message}">`;
+};
+
+// Fetch an image for selected breed
+const getDogImg = async () => {
+    const url = `https://dog.ceo/api/breed/${dogSelector.value}/images/random`;
+
+    await fetch(url) 
```

Comment:
you need async await only if you want to get value outside from fetch (const data = fetch(.......))
So here you can remove async from function and await from fetch.